apps/condo/domains/billing/access/BillingCategory.js

```diff
@@ -0,0 +1,30 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema billing.BillingCategory 'name:Text'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadBillingCategories ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return {}
+}
+
+async function canManageBillingCategories ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
```

Comment:
are you sure that support will change it? is it not an integration responsibility?

Reply:
As of now, i don't want to add add ability to create or update categories to integrations. This would be hard to manage